zerver/views/registration.py

```diff
@@ -1138,7 +1137,7 @@ def find_account(
             for email in raw_emails.split(","):
                 try:
                     validators.validate_email(email)
-                    emails.append(email)
+                    emails.append(email)  # nocoverage
```

Comment:
I think we lost coverage because of removing some test which we shouldn't have. Let's fix this.

Reply:
I think we can remove the whole snippet. I wasn't able to understand the case when the block was executed

Reply:
> I wasn't able to understand the case when the block was executed

When the URL has email as query params, this was executed -- after POST request to list down emails. We no longer need this.